frappe/email/__init__.py

```diff
@@ -12,7 +12,9 @@ def get_contact_list():
 	"""Returns contacts (from autosuggest)"""
 
 	try:
-		out = filter(None, frappe.db.sql_list("""select distinct email_id from `tabContact`"""))
+		out = frappe.db.sql_list("""select concat(email_id, " < ", concat(first_name,
```

Comment:
Query should return email and full name, the concatenation should happen at the client side.